lox.js

```diff
@@ -84,6 +85,8 @@ app.delete('/lock/:lockId', function(req, res) {
   });
 });
 
+lua.initialize();
```

Comment:
I _think_ this might be unnecessary. If you just do your synchronous init logic in the main line of the lua module then i think node guarantees this to be run before `require` returns.